DigitalLearningSolutions.Web/ViewModels/Register/SummaryViewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Register
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+
+    public class SummaryViewModel: IValidatableObject
+    {
+        public string? FirstName { get; set; }

```

Comment:
Can these be non-optional? (I'd expect these to always exist in summary page and we could set values in the constructor on page load)